src/screens/DiscoveryScreenV2/fragment/UsersFragment.js

```diff
@@ -25,7 +25,6 @@ const UsersFragment = ({
   isLoadingDiscoveryUser = false,
   isFirstTimeOpen,
   initialUsers = [],
-  setInitialUsers = () => {},
   followedUsers = [],
   setFollowedUsers = () => {},
   unfollowedUsers = [],
```

Comment:
<img src="https://avatars.githubusercontent.com/in/347564?s=41" alt="Image description" width="20" height="20">   CodeRabbit

The `setInitialUsers` function has been removed in the new hunk. If this function is not used anywhere else, then it's fine. However, if it's used elsewhere, removing it might cause issues.



<!-- This is an auto-generated comment by OSS CodeRabbit -->